### ISSUE TITLE
Is multi-scale training implemented?

### ISSUE THREAD

**Issue by JesseYang** [leetenki/YOLOv2#9]
As shown in the paper, the multi-scale training can import the mAP on VOC 2007 test set by 1.4. Has multi-scale training been implemented by YOLOv2?

**Comment by aliceyayunji** [comment]
Yes, leetenki implemented the  multi-scale training, see the yolov2_train.py file.
